prefetch.bs

```diff
@@ -323,6 +323,12 @@ These algorithms are based on [=process a navigate fetch=].
         1. If |partitionKey| is not equal to |proposedPartitionKey|, then return "`Blocked`".
 
             <div class="issue">It might be possible to "downgrade" to [=uncredentialed prefetch=] in this case.</div>
+        1. If |request|'s [=request/URL=] is not [=potentially trustworthy URL|potentially trustworthy=], then return "`Blocked`".
+
+            <div class="note">This is intended to both reduce the likelihood of prefetch traffic being visible to an on-path attacker, and to encourage the use of cryptographic schemes over public networks.</div>
+        1. If |request| cannot be fetched given |anonymizationPolicy| for an [=implementation-defined=] reason, then return "`Blocked`".
+
+            <div class="note">This explicitly acknowledges that implementations might have additional restrictions. For instance, anonymized traffic might not be possible to some hosts, such as those that are not publicly routable and those that have <a href="https://buettner.github.io/private-prefetch-proxy/traffic-advice.html">traffic advice</a> declining private prefetch traffic.
```

Comment:
IMO we should make traffic advice normative at some point?

Reply:
Maybe.

On the one hand, consistency is good.

On the other hand, it's more descriptive of the proxy than of the web browser, and it's possible we'd get bogged down in the details of how anonymization happens in the process.